### ISSUE TITLE
Update seems to fail from v0.8.10 on Mac

### ISSUE THREAD

**Issue by Gozala** [beakerbrowser/beaker#1566]
**Describe the bug**

**To Reproduce**
1. Go to 'settings'
2. Click on 'Check for beta releases'
3. Starts downloading 
4. Fails with error shown on the screen

**Expected behavior**

**Screenshots**
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/21236/81979424-cbf17280-95e1-11ea-91e2-0a7a865fad90.png">

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.4
 - Browser: Beaker
 - Version: v0.8.10

**Comment by pfrazee** [comment]
Yeah really sorry about this -- we had to change the appid to the company from my name (pfrazee -> bluelinklabs) and that broke the auto update. You can update by downloading the dmg: https://github.com/beakerbrowser/beaker/releases/download/1.0.0-prerelease.1/beaker-browser-1.0.0-prerelease.1.dmg

**Comment by Gozala** [comment]
No worries, just wanted to make sure you were aware
